docs/developing/secrets_details.md

```diff
@@ -0,0 +1,40 @@
+---
+
+copyright:
+years: 2021
+lastupdated: "2021-08-16"
+
+---
+
+{:shortdesc: .shortdesc}
+{:new_window: target="_blank"}
+{:codeblock: .codeblock}
+{:pre: .pre}
+{:screen: .screen}
+{:tip: .tip}
+{:download: .download}
+
+# Secrets Manager details
+{: #secrets_details}
+
+The Secrets Manager provides secure storage for sensitive information like authentication credentials or encryption keys. These secrets are securely deployed by {{site.data.keyword.ieam}} so that only the services configured to receive a secret will have access to it. The [Hello Secret World Example ![Opens in a new tab](../images/icons/launch-glyph.svg "Opens in a new tab")](https://github.com/open-horizon/examples/blob/master/edge/services/helloSecretWorld/CreateService.md) provides an overview of how to exploit secrets in an edge service.
+
+Currently, {{site.data.keyword.ieam}} supports the use of Hashicorp Vault as the Secrets Manager. Secrets created using the hzn CLI are mapped to Vault secrets using the KV V2 Secrets Engine. This means that the details of every {{site.data.keyword.ieam}} secret are composed of a key and a value. Both are stored as part of the details of the secret, and both can be set to any string value. A common usage of this feature is to provide the type of secret on the key and sensitive information as the value. For example, set the key to 'basicauth' and set the value to 'user:password'. In so doing, the service developer can interrogate the type of secret, enabling the service code to handle the value correctly.
+
+The names of secrets in the Secrets Manager are never known by a service implementation. It is not possible to convey information from the vault to a service implementation using the name of a secret.
```

Comment:
Is it vault or Vault?

Reply:
I don't know. Hashicorp's secrets vault is called Vault, so I thought I should refer to it with a capital V.